app/api/career/[_id]/route.ts

```diff
@@ -53,17 +61,19 @@ const DELETE = async (
 ) => {
   try {
     await connectMongoDB();
-    const dataToDelete = await Hiring.findByIdAndUpdate(
+
+    const JobToDelete = await Hiring.findByIdAndUpdate(
```

Comment:
`isActive` is different from deletion, it's just to make it as closed. Maybe let's rename this to `closeJobOpening`, and for deletion, let's introduce a new `isDeleted` to the model, and set it to `true` for deletion, and in the fetch, we exclude `isDeleted` data